src/main/resources/org/jenkinsci/plugins/github/admin/GitHubHookRegisterProblemMonitor/message.groovy

```diff
@@ -1,11 +1,24 @@
 package org.jenkinsci.plugins.github.admin.GitHubHookRegisterProblemMonitor
 
+import hudson.util.VersionNumber
+import jenkins.model.Jenkins
+
 def f = namespace(lib.FormTagLib)
 
-div(class: 'alert alert-warning') {
-    form(method: 'post', action: "${rootURL}/${my?.url}/act", name: my?.id) {
-        f.submit(name: 'yes', value: _('view'))
-        f.submit(name: 'no', value: _('dismiss'))
+if (Jenkins.getVersion().isNewerThan(new VersionNumber("2.88"))) {
+    div(class: 'alert alert-warning') {
+        form(method: 'post', action: "${rootURL}/${my?.url}/act", name: my?.id) {
+            f.submit(name: 'yes', value: _('view'))
+            f.submit(name: 'no', value: _('dismiss'))
+        }
+        text(_('hook.registering.problem'))
+    }
+} else {
+    div(class: 'warning') {
+        form(method: 'post', action: "${rootURL}/${my?.url}/act", name: my?.id) {
+            text(_('hook.registering.problem'))
+            f.submit(name: 'yes', value: _('view'))
+            f.submit(name: 'no', value: _('dismiss'))
+        }
```

Comment:
Why do you want to put `text` to `form`?
(and not for Jenkins >= 2.88?)

Reply:
Originally `text(_('hook.registering.problem'))` was invoked after the form clause. It should be restored if there is no specific reason to do that

Reply:
@oleg-nenashev No, there is an important reason for that. The DOM generated should be synced with the proposal on https://github.com/jenkinsci/jenkins/pull/2857

Reply:
This `else` block is for Jenkins <= 2.88, that is, without jenkinsci/jenkins#2857 .
Do you mean this DOM should be fixed regardless of jenkinsci/jenkins#2857 ?

Reply:
@ikedam Yes, https://github.com/jenkinsci/jenkins/pull/2857 provides a new re-styling and if a maintainer wants to use it, needs to perform a small change like this. But we want to keep backward compatibility in the UI.

Reply:
@oleg-nenashev Please, could you update your review?

Reply:
@recena 
I'm afraid you didn't get the point I asked.
I suppose this PR contains 2 changes:

* Fix for misplaced text and buttons. (this means there's a bug in the current version)
* Preparation for jenkinsci/jenkins#2857 .

You haven't mentioned the prior one, and it makes me wonder whether that change is expected one, or the one by mistake. 

Attaching screenshot might clarify the fix by this PR.

Reply:
> No, there is an important reason for that. The DOM generated should be synced with the proposal on ...

It is for the legacy branch. Why does it need to be synced?

Reply:
@oleg-nenashev I don't know what you are not understanding. There are two blocks, one with the original implementation and other with the new one. And yes, the order on how the elements are rendered is important.